dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
> That's what we agreed on in our standup. 

We agreed to have this, not that it should be with the style linters. Or at least I din't understand that. :confused: 

> Where would you put it?

in a different test suite called something like `database structure`?

> And does it really matter?

It would be run even if you put it in the wrong place, but ordering things is important :unamused: